src/main/java/hillel/spring/doctor/domain/Review.java

```diff
@@ -0,0 +1,71 @@
+package hillel.spring.doctor.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+public class Review {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @Version
+    private Integer version;
+
+    private Integer doctorRecordId;
+    private LocalDateTime reviewDate;
+
+    private Byte serviceRating;
+    private Byte equipmentRating;
+    private Byte qualificationRating;
+    private Byte treatmentRating;
+    private Byte totalRating;
```

Comment:
I wouldn't recommend using anything but Integer here.

For example imagine we have a `Map<Byte,  String> numberToWord` and you know that it contains bytes from 0 to 16 with values "zero", "one"... "sixteen" respectively. What do you expect for `numberToWord.get(5)`?
If your answer is "five" then just use an `Integer` in this entity.
If you know why it is a `null` - go ahead and just skip this comment.